File: MPI/c/hello.c
#include <stdio.h>
#include <mpi.h>

int main(int argc, char **argv)
{
  int ierr;

  // Create child processes (# specified at runtime)
  ierr = MPI_Init(&argc, &argv);
  printf("Hello world\n");

  // Stop this process
  ierr = MPI_Finalize();
}

File: MPI/fortran/hello.f
c     A basic 'Hello World' program

      program helloworld
    
      include '/usr/lib/x86_64-linux-gnu/openmpi/include/mpif.h'

      integer ierr  ! error status for initializing MPI

      call MPI_INIT ( ierr ) 
      print *, "Hello world!"
      call MPI_FINALIZE ( ierr )

      stop
      end
